sickbeard/webapi.py

```diff
@@ -273,6 +275,8 @@ class ApiCall(ApiHandler):
 
     def __init__(self, args, kwargs):
         # missing
+        super(ApiCall, self).__init__(*args, **kwargs)
+
```

Comment:
Did you try this?
I wanted to added it a while back, but it introduced some issues. Can remember which one though.